File: Week 1 - C/termsAndConditions.c
#include <stdio.h>

int main (void){

    char c;

    printf("Do you accept terms and conditions? [Y/N]: ");
    scanf("%c", &c);

    if (c == 'y' || c == 'Y'){
        printf("Ok, we good then, you can enter");
    }
    else if (c == 'n' || c == 'N'){
        printf("You shall not pass");
    }
    else{
        printf("Invalid option");
    }
}

File: Week 6 - Python/termsAndConditions.py
option = input("Do you accept terms and conditions? [Y, N]: ")

# In python, for logical condition we use the literal words "or", "and", "not"
if option == "Y" or option == "y":
    print("Ok, we good then, you can enter")
elif option == "N" or option == "n":
    print("You shall not pass")
else:
    print("Invalid option")

# The same but with using a list of options
option = input("Do you accept terms and conditions? [Y, N]: ")
if option in ["Y", "y", "yes", "Yes", "yeah"]:
    print("Ok, we good then, you can enter")
elif option in ["N", "n", "no", "No", "nah"]:
    print("You shall not pass")
else:
    print("Invalid option")